### PR TITLE
Add Color.to_int to convert to the usual int sRGB representation

### PR DESCRIPTION
**Pull request by Chris00**

### CODE REVIEW

**Comment by dbuenzli** [comment]
I'm not really convinced by adding this function. The problem is that there are too many ints (`int`, `int32`, `int64`) types and layouts (rgb, rgba, argb, bgr, bgra, abgr) you'd like to possibly have. One thing that I'd be willing to integrate though is the counter part to `v_srgbi`, i.e:

``` ocaml
val to_srgbi : color -> (int * int * int * float)
```

**Comment by Chris00** [comment]
My purpose was mainly to be able to easily use these colors with the `Graphics` module (or `Cairo`,...).  `to_srgbi` will help so I updated this PR.

File: src/gg.mli [hunk 1]
```diff
@@ -3127,13 +3127,13 @@ module Color : sig
       a color value. *)
 
   val v_srgb : ?a:float -> float -> float -> float -> color
-  (** [v r g b ~a] is the sRGB color [(r, g, b, a)] converted to a color
-      value. *)
+  (** [v r g b ~a] is the {e non-linear} sRGB color [(r, g, b, a)]
+      converted to a (linear) [Gg] color value. *)
```
Comment (dbuenzli) [inline src/gg.mli:3131]: I'm not very keen on all these `non-linear` and `linear` you added. sRGB is by definition non-linear so it's more confusing than enlighting to say non-linear sRGB.

Comment (dbuenzli) [reply src/gg.mli:3131]: I'll merge and amend the PR. Sorry for having missed that for the release.

**Comment by dbuenzli** [comment]
Thanks ! Your (amended) patch is in as a1cd31593c744c6a2473e81a4fb.

**Pull request closed without merging**